haiku-games/2048/2048-1.0.0.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Clone of the game 2048"
+DESCRIPTION="Clone of the popular 2048 game.
+
+HOW TO PLAY: Use your arrow keys to move the tiles.
```

Comment:
I'd remove the repeat of the SUMMARY and go with:
"The goal of this popular tile game is get to tile 2048.
Use the cursor keys to move the tiles. When two tiles with the same number \
touch, they merge into one."